logging/config/cluster-logging-tolerations.adoc

```diff
@@ -96,4 +96,10 @@ include::modules/cluster-logging-collector-tolerations.adoc[leveloffset=+1]
 [id="cluster-logging-tolerations-addtl-resources"]
 == Additional resources
 
+
+ifdef::openshift-enterprise,openshift-origin[]
 * xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations[Controlling pod placement using node taints].
+endif::[]
+ifdef::openshift-rosa,openshift-osd[]
```

Comment:
s/openshift-osd/openshift-dedicated

Reply:
The link is broken on the ROSA preview (points to OCP docs and just needs to be swapped to ROSA and OSD xrefs)